jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/wizard/SamInitSelectionPanel.kt

```diff
@@ -147,7 +147,7 @@ class SamInitSelectionPanel(
      * Updates UI fragments in the wizard after a combobox update
      */
     private fun wizardUpdate() {
-        val selectedRuntime = runtimeComboBox.selectedItem as? Runtime
+        val selectedRuntime = runtimeComboBox.selectedItem as? LambdaRuntime
```

Comment:
If you go through the model (`runtimes`), you dont need the cast

Reply:
Changed